### PR TITLE
Document enums for the callRecords subnamespace

### PR DESCRIPTION
**Pull request by GageAmes**
Add a enums file for the callRecords subnamespace and reference the enum names as the type in the properties table of the parent resource topic.

### CODE REVIEW

**Comment by GageAmes** [comment]
> Looks good to me. I'll loop in Millicent in your email to verify the use of fully qualified type vs partially qualified type in the "Type" column of the properties table.

Thanks, @angelgolfer-ms.  I opted to use fully qualified type since that's what was done [elsewhere](https://github.com/microsoftgraph/microsoft-graph-docs/commit/cd292f5823885d90aad04674b7390ec234515de4#diff-e245b161246ee4bb8eb18fdc1b7c14f0R20).  I can change them from `microsoft.graph.callRecords.` to `callRecords.` if you prefer, though.

**Comment by millicentachieng** [comment]
Either `microsoft.graph.callRecords.` or just `callRecords.` should work just fine.

File: api-reference/beta/resources/callrecords-callrecord.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ Represents a single peer-to-peer call or a group call between multiple participa
 |id|String|Unique identifier for the call record. Read-only.|
 |joinWebUrl|String|Meeting URL associated to the call. May not be available for a peerToPeer call record type.|
 |lastModifiedDateTime|DateTimeOffset|UTC time when the call record was created. The DatetimeOffset type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`|
-|modalities|string collection|List of all the modalities used in the call. Possible values are: `unknown`, `audio`, `video`, `videoBasedScreenSharing`, `data`, `screenSharing`, `unknownFutureValue`.|
+|modalities|microsoft.graph.callRecords.modality collection|List of all the modalities used in the call. Possible values are: `unknown`, `audio`, `video`, `videoBasedScreenSharing`, `data`, `screenSharing`, `unknownFutureValue`.|
```
Comment (Lauragra) [inline api-reference/beta/resources/callrecords-callrecord.md:31]: Global: Please make this a link to the resource.

Comment (GageAmes) [reply api-reference/beta/resources/callrecords-callrecord.md:31]: @Lauragra what resource?  `microsoft.graph.callRecords.modality` is an enum.

Comment (GageAmes) [reply api-reference/beta/resources/callrecords-callrecord.md:31]: According to https://msgo.azurewebsites.net/add/document/guidelines/enums-in-subnamespaces.html#first-option-in-a-general-enums-file-for-that-subnamespace it states "Do NOT provide a link to the enumeration type."  @angelgolfer-ms could you clarify?

**Pull request merged**